### ISSUE TITLE
Gun/Crosshairs only shoot/aim bottom left corner of Screen in 4k

### ISSUE THREAD

**Issue by handsome-van** [argonlefou/HotdRemake_ArcadePlugin#4]
Hello,

I'm using the new 2-Player Arcade patch which is awesome, thank you. I have only 1 issue though. I can only play the game in 1080p resolution. My screen is 4k (4096x2160) and when I select that resolution in the Arcade Patch, the game will start but the gun/crosshairs on both of my Gun4IR's will only aim and shoot in the bottom left quarter of the screen. Even though I'm aiming in the upper right corner of the screen, the farthest gun will shoot is in the middle. 

Now if I take the resolution down to 1920x1080 in the patch, both guns can aim and shoot of all 4 corners of the screen no problem, works perfect. But the game doesn't look as good as it does when I'm in 4k obviously. I have a strong rig (Rysen5600x, NVIDIA GeForce 3070, 32g RAM) so I don't think that would be the problem.

Any help would be greatly appreciated.

Thanks!!

**Comment by handsome-van** [comment]
Thank you getting back to me. So I found out the problem...most games recognize if Windows scaling is not 100% and corrects it when an application is running. HOTD Remake unfortunately does not. Because my Arcade setup is with a 4k 50-inch screen I have the scaling set at 225% so I can see text, icons, etc and they are not super small.

What I did to fix the problem is add in the AHK script to change the DPI to 100% (96) when the game is started, and then to change the DPI back to 225% (216) when I exit the game and close the AHK script. Problem solved.

Once again thank you very much for your response in a timely manner. Keep up the AMAZING WORK with all the cool Light Gun stuff you are doing!! Love it!! -Cheers
